### ISSUE TITLE
Container crashes on Command/Reset message

### ISSUE THREAD

**Issue by chrniem** [Spruik/PackML-MQTT-Simulator#46]
Container is started with docker compose as follows:

```
services:
  packml-ac-w1-l2:
    image: ghcr.io/libremfg/packml-simulator
    environment:
      SITE: HeinzMeierGmbH/Aachen
      AREA: Werk1
      LINE: Line1/Cell2
      MQTT_URL: mqtt://192.168.113.110
      MQTT_PORT: 1883
    mem_limit: 30MB
```

```
2024-10-24T11:34:05.042Z | info: HeinzMeierGmbH/Aachen/Werk1/Line1/Cell2/Admin/ProdProcessedCount/0/Count : 0
2024-10-24T11:34:05.043Z | info: HeinzMeierGmbH/Aachen/Werk1/Line1/Cell2/Admin/ProdProcessedCount/0/AccCount : 0
2024-10-24T11:34:10.020Z | info: HeinzMeierGmbH/Aachen/Werk1/Line1/Cell2/Status/StateCurrentStr : Clearing
2024-10-24T11:34:10.020Z | info: HeinzMeierGmbH/Aachen/Werk1/Line1/Cell2/Status/StateCurrent : 4
2024-10-24T11:34:13.256Z | info: HeinzMeierGmbH/Aachen/Werk1/Line1/Cell2/Status/StateCurrentStr : Stopped
2024-10-24T11:34:13.257Z | info: HeinzMeierGmbH/Aachen/Werk1/Line1/Cell2/Status/StateCurrent : 2
/machine/index.js:148
    stateCommand(topic, message, state)
    ^
ReferenceError: stateCommand is not defined
    at Client.<anonymous> (/machine/index.js:148:5)
    at Client.emit (node:events:519:28)
    at MqttClient.<anonymous> (/machine/clients/mqtt.js:40:18)
    at MqttClient.emit (node:events:519:28)
    at handlePublish (/machine/node_modules/mqtt/build/lib/handlers/publish.js:97:20)
    at handle (/machine/node_modules/mqtt/build/lib/handlers/index.js:28:35)
    at work (/machine/node_modules/mqtt/build/lib/client.js:227:40)
    at writable._write (/machine/node_modules/mqtt/build/lib/client.js:252:13)
    at writeOrBuffer (/machine/node_modules/readable-stream/lib/internal/streams/writable.js:334:12)
    at _write (/machine/node_modules/readable-stream/lib/internal/streams/writable.js:283:10)
Node.js v20.18.0
```

Thanks

**Comment by tomhollingworth** [comment]
Confirmed.

**Comment by tomhollingworth** [comment]
Hi @chrniem, this issue is now resolved, can you please use >= v2.0.7 by either:

```diff
services:
  packml-ac-w1-l2:
-    image: ghcr.io/libremfg/packml-simulator
+    image: ghcr.io/libremfg/packml-simulator:latest
+    pull_policy: always
    environment:
      SITE: HeinzMeierGmbH/Aachen
      AREA: Werk1
      LINE: Line1/Cell2
      MQTT_URL: mqtt://192.168.113.110
      MQTT_PORT: 1883
    mem_limit: 30MB
```

or 


 ```diff
services:
  packml-ac-w1-l2:
-    image: ghcr.io/libremfg/packml-simulator
+    image: ghcr.io/libremfg/packml-simulator:v2.0.7
    environment:
      SITE: HeinzMeierGmbH/Aachen
      AREA: Werk1
      LINE: Line1/Cell2
      MQTT_URL: mqtt://192.168.113.110
      MQTT_PORT: 1883
    mem_limit: 30MB
```

**Comment by chrniem** [comment]
Works now, thanks.
